python/paddle/fluid/tests/unittests/test_eager_deletion_transformer.py

```diff
@@ -15,13 +15,12 @@
 import os
 import unittest
 os.environ['FLAGS_eager_delete_tensor_gb'] = "0.0"
+# os.environ['FLAGS_fast_eager_deletion_mode'] = "1"
```

Comment:
If line18 is unnecessary, it should be removed.

Reply:
Done.